test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/conftest.py

```diff
@@ -80,6 +80,13 @@
     "il-central-1",
 ]
 
+# TODO: Expand this list
+G5_AVAILABLE_REGIONS = [
+    "ca-central-1",
+    "us-west-2",
```

Comment:
us-east-1 is in SM execution regions - should probably be added here too to avoid false positives. Same comment applies to PT change

Reply:
Yes, that's a good catch, will add it here.

Reply:
Done, thanks!